### DISCUSSION TITLE
Replacement of Service Account

### DISCUSSION THREAD

**Discussion by JoyWorksToo** [dataplat/dbatools#7682] [Ideas]
Lately we have been replacing an old domain and with that migration of Service Accounts to be domain.

Getting all the disk permissions in place has been a challenge. Is there a command to assist? 

Also of note: export-dbalogin has been great for within SQL but it missed one entry as owner of AG Endpoint.

**Comment by niphlod** [comment]
"all the disk permissions in place" : did you grant specific perms to some specific user ? Late mssql version use a virtual account in the ACLs, so that you can later just change the service account via SMO and everything continues to work out of the box.
We do not have anything that scans all folders and looks for ACLs and sets them appropriately, it'd be quite a nightmare supporting each "ACL way of doing things" for everybody.
